### ISSUE TITLE
Connection refused by the server

### ISSUE THREAD

**Issue by jauzzz** [miguelgrinberg/python-socketio#483]
I'm running a docker container for my [socketio-server](https://github.com/jauzzz/websocket)
In the project, there is also a [test script](https://github.com/jauzzz/websocket/blob/master/websocket/liveroom/tests/test_join.py) (used to test the performance of concurrency)

The problem is that, the results of the docker test are inconsistent with the results of the local test
- testing local, 2000 client connect is fine
- testing docker, 1200 client connect is fine
- testing docker, 2000 client connect, cause a large number of problem `Connection refused by the server`
![2000 client test result](https://tva1.sinaimg.cn/large/007S8ZIlgy1gepgv5lkmuj30ql0d3af4.jpg)

So I monitored the tcp connection, find out that tcp overflowed.
![image](https://user-images.githubusercontent.com/17628634/81632627-59479200-943d-11ea-93ea-8767177de994.png)

I configued docker-compose sysctls arguments, `net.core.somaxconn=8192` , `net.ipv4.tcp_max_syn_backlog=2048`, and it works! 

However, the application seems no change (run at 8200 port).
![image](https://user-images.githubusercontent.com/17628634/81632858-d96df780-943d-11ea-99bf-9de108ecbb77.png)

In summary, here is my question:
- Is there a backlog argument for socketio-server
- what might the reason for `Connection refused by the server`

**Comment by miguelgrinberg** [comment]
I think you misunderstand what part python-socketio plays in all of this. This is just the application layer. The problems and questions that you have apply to the web server that you are using, not to python-socketio. This project does not implement the server or anything having to do with networking.

**Comment by jauzzz** [comment]
> I think you misunderstand what part python-socketio plays in all of this. This is just the application layer. The problems and questions that you have apply to the web server that you are using, not to python-socketio. This project does not implement the server or anything having to do with networking.

thanks for your reminder
